File: src/main/java/org/team1/nbe1_3_team01/domain/group/controller/request/TeamApprovalUpdateRequest.java
package org.team1.nbe1_3_team01.domain.group.controller.request;

import jakarta.validation.constraints.NotNull;
import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class TeamApprovalUpdateRequest {

    @NotNull(message = "teamId가 누락되었습니다.")
    private Long teamId;

}

File: src/main/java/org/team1/nbe1_3_team01/domain/group/controller/request/TeamApprovalUpdateRequest.kt
package org.team1.nbe1_3_team01.domain.group.controller.request

import jakarta.validation.constraints.NotNull
import lombok.AllArgsConstructor
import lombok.Data

data class TeamApprovalUpdateRequest(
    @field:NotNull(message = "teamId가 누락되었습니다.")
    val teamId: Long
) {
}
